scripts/tmux-user-shell.sh

```diff
@@ -0,0 +1,41 @@
+sleep 2
```

Comment:
Perhaps a comment to explain why it sleeps here.

Reply:
Done